From 93b27ef56cc81fa21eafbbd38b32319eac859fcb Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:30:56 -0400
Subject: [scsi] hpsa: add MSA 2040 to list of external target devices

Message-id: <1410186734-19986-6-git-send-email-thenzl@redhat.com>
Patchwork-id: 92023
O-Subject: [RHEL6.6 e-stor PATCH 05/83] hpsa: add MSA 2040 to list of external target devices
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

Signed-off-by: Scott Teel <scott.teel@hp.com>
Acked-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit e06c8e5c34498d1ea0468401012be8d1c853c585)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index bb848af..f012782 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -1792,6 +1792,7 @@ static unsigned char *ext_target_model[] = {
  "MSA2312",
  "MSA2324",
  "P2000 G3 SAS",
+ "MSA 2040 SAS",
  NULL,
 };
 
-- 
1.7.1